src/javascripts/components/staff/showStaff.js

```diff
@@ -1,14 +1,21 @@
+import filterStaff from './filterStaffList';
+
 const showStaff = (staffArray, user) => {
-  document.querySelector('#view').innerHTML = '';
-  document.querySelector('#stage').innerHTML = '';
-  document.querySelector('#filter-container').innerHTML = '';
```

Comment:
Believe you need this on showStaff.js 
`document.querySelector('#stage').innerHTML = '';`